jbcli/jbcli/cli/jb.py

```diff
@@ -135,10 +135,11 @@ def add(applications, add_desktop):
                                 'Failed to add {} to Github Desktop.'.format(
                                     app))
                 try:
-                    dockerutil.run(
-                        '/venv/bin/python manage.py loadjuiceboxapp {}'.format(
-                            app))
-                    echo_success('{} was added successfully.'.format(app))
+                    if not jbapiutil.load_app(app):
```

Comment:
Can we also utilize this here? https://github.com/juiceinc/devlandia/blob/3937becc0a3e61b649ad0406878c50d9dae57c34/jbcli/jbcli/utils/dockerutil.py#L36

Reply:
Yup, I wanted to wait until we got yours merged.